src/Entity/ProductOption.php

```diff
@@ -103,6 +111,24 @@ public function getElementType() {
     return $this->elementType;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getProductTypes() {
+    if ($this->productTypes) {
```

Comment:
Really minor change, but can we refactor this function so that we return right away `if (!$this->productTypes)`. I think it will be a bit more readable that way.